reference/6/Microsoft.PowerShell.Core/Set-PSSessionConfiguration.md

```diff
@@ -51,76 +51,95 @@ Set-PSSessionConfiguration [-Name] <String> [-RunAsCredential <PSCredential>]
+```powershell
+Register-PSSessionConfiguration -Name "AdminShell" -AssemblyName "C:\Shells\AdminShell.dll" -ConfigurationTypeName "AdminClass"
+Set-PSSessionConfiguration -Name "AdminShell" -StartupScript "AdminConfig.ps1"
+Set-PSSessionConfiguration -Name "AdminShell" -StartupScript $Null
 ```
 
 ### Example 3: Display results
 
+This example increases the value of the **MaximumReceivedObjectSizeMB** property to 20. This command
+also prompts you to restart the **WinRM** service. The change is not effective until the **WinRM**
+service is restarted.
+
+```powershell
+Set-PSSessionConfiguration -Name "IncObj" -MaximumReceivedObjectSizeMB 20
 ```
-PS C:\> Set-PSSessionConfiguration -Name "IncObj" -MaximumReceivedObjectSizeMB 20
+
+```Output
 WSManConfig: Microsoft.WSMan.Management\WSMan::localhost\Plugin\IncObj\InitializationParameters
+
 ParamName                       ParamValue
 ---------                       ----------
 psmaximumreceivedobjectsizemb   20
+
 "Restart WinRM service"
 WinRM service need to be restarted to make the changes effective. Do you want to run the command "restart-service winrm"?
 [Y] Yes  [N] No  [S] Suspend  [?] Help (default is "Y"): y
 ```
 
-The **Set-PSSessionConfiguration** command in this example increases the value of the **MaximumReceivedObjectSizeMB** property to 20.
+### Example 4: Display results in different ways
 
-The **Set-PSSessionConfiguration** command returns a **Microsoft.WSMan.Management.WSManConfigLeafElement** object that shows the parameter name and new value.
+In this example, `Set-PSSessionConfiguration` changes the startup script in the **MaintenanceShell**
+session configuration to `Maintenance.ps1`. The output shows the change and prompts you to restart
+the **WinRM** service. The response is "y" (yes).
 
-This command also prompts you to restart the **WinRM** service.
-The change is not effective until the **WinRM** service is restarted.
-
-### Example 4: Display results in different ways
+`Get-PSSessionConfiguration` gets the **MaintenanceShell** session configuration. The pipeline
+operator (|) sends the results of the command to `Format-List`, which displays all the properties of
+the configuration object in a list. Next, using the WSMan provider, we view the initialization
+parameters for the **MaintenanceShell** configuration. `Get-ChildItem` (alias `dir`) gets the child
+items in the **InitializationParameters** node for the **MaintenanceShell** plug-in. For more
+information about the WSMan provider, type `Get-Help wsman`.
 
 ```
-The first command uses the **Set-PSSessionConfiguration** cmdlet to change the startup script in the MaintenanceShell session configuration to Maintenance.ps1. The output of this command shows the change and prompts you to restart the **WinRM** service. The response is "y" (yes).
 PS C:\> Set-PSSessionConfiguration -Name "MaintenanceShell" -StartupScript "C:\ps-test\Maintenance.ps1"
```

Comment:
Cleaned up the prompt in the other doc, should clean these up, too.

Reply:
fixed